### ISSUE TITLE
New splash screen

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#653]
@amazingphilippe @katewilhelm we are unable to have the splash screen load for very long for various reasons.
It is recommended that we have a transition screen before launching users into onboarding so that there is an orientation for all users. We were unable to verify with alt text the splash image or the canada.ca watermark for those technical constraints.

**Comment by JuliannaR** [comment]
Here is the new splash screen for the app. You can find this in the updated V1 figma. 
<img width="235" alt="Screen Shot 2020-07-15 at 11 08 14 AM" src="https://user-images.githubusercontent.com/35537308/87562123-96bddb00-c68b-11ea-929c-4fb2ac535569.png">
